UDV-Core/src/Extensions/Contribute/src/UpdateDocument.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * Class: UpdateDocument
+ * Description :
+ * The UpdateDocument is an object handling the creation view
+ *
+ */
+
+ /**
+  *
+  * @constructor
+  * @param { HTML DOM Element object } creationContainer
+  * @param { contributeController } contributeController
+  */
+
+export function UpdateDocument(updateContainer, contributeController){
+
+  this.contributeController = contributeController;
+  this.updateContainer = updateContainer;
+  this.browser = this.contributeController.documentController.documentBrowser;
+  this.data; //current document data (document shown in the browser)
+
+  this.schemaType = { //schema defining the update form, using Alpaca
+    "type": "object",
+    "properties": {
+      //will be set by calling docModelToSchema
+    }
+  };
+
+  this.optionsUpdate = { //options to render the update form
+    "form": {
+      "attributes":{
+        "id":"updateForm"
+      }
+    },
+    "fields":{
+      //will be set by calling docModelToSchema
+    }
+  };
+
+  this.initialize = function initialize()
+  {
+    var docUpdateButton = document.createElement('button');
+    docUpdateButton.id = "docUpdateButton";
+    var text = document.createTextNode("Update");
+    docUpdateButton.appendChild(text);
+    document.getElementById("browserWindowTabs").appendChild(docUpdateButton);
+
+    var docDeleteButton = document.createElement('button');
+    docDeleteButton.id = "docDeleteButton";
+    var text = document.createTextNode("Delete");
+    docDeleteButton.appendChild(text);
+    document.getElementById("browserWindowTabs").appendChild(docDeleteButton);
```

Comment:
Please retrieve "browserWindowTab" from e.g. documentController attribute instead of harcoding it.

Reply:
ok